lib/compute/agent-nodes.ts

```diff
@@ -145,8 +147,20 @@ export class AgentNodes {
       remoteUser: 'Administrator',
       maxTotalUses: -1,
       minimumNumberOfSpareInstances: 2,
+      numExecutors: 2,
+      amiId: 'ami-006c911a4bd898d4a',
+      initScript: 'echo',
+      remoteFs: 'C:\\Users\\Administrator\\jenkins',
+    };
+    this.WINDOWS2019_X64_GRADLE_CHECK = {
+      agentType: 'windows',
+      workerLabelString: 'Jenkins-Agent-Windows2019-X64-C524xlarge-Single-Host',
+      instanceType: 'C54xlarge',
+      remoteUser: 'Administrator',
+      maxTotalUses: 1,
+      minimumNumberOfSpareInstances: 1,
       numExecutors: 1,
-      amiId: 'ami-097c07673f792f89e',
+      amiId: 'ami-006c911a4bd898d4a',
```

Comment:
Can you confirm the AMI is public?

Reply:
I would have to disable the old one once deployment happens or the quota will stop it.